bundles/org.openhab.binding.ecovacs/pom.xml

```
@@ -14,13 +14,20 @@
 
   <name>openHAB Add-ons :: Bundles :: Ecovacs Binding</name>
   <properties>
-    <smack.version>4.3.3</smack.version>
+    <smack.version>4.5.0-beta2</smack.version>
```

There is a newer beta:

```suggestion
    <smack.version>4.5.0-beta5</smack.version>
```

Java 8 support was dropped so you should use `smack-java11` now.